composer.json

```
@@ -87,7 +87,7 @@
         "symfony/console": "4.4.* || 5.2.*",
         "symfony/debug": "4.4.* || 5.2.*",
         "symfony/debug-bundle": "4.4.* || 5.2.*",
-        "symfony/dependency-injection": "4.4.* || 5.2.*",
+        "symfony/dependency-injection": "^4.4.20 || ^5.2.4",
```

```suggestion
        "symfony/dependency-injection": "~4.4.20 || ~5.2.4",
```

As we explicitly want to disallow `5.3.X`

---

Thanks, yes! Adjusted in 832b16e205dab5fa1155fa0b1cb3ef35450170d3.